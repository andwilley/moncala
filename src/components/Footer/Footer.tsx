import React from "react";

export const Footer: React.FC = () => {
  return (
    <div className="Footer h2">
      <p className="mt2 f7 fr">Created By: Drew Willey. For: Mon Cala Project Application.</p>
    </div>
  );
};

export default Footer;
