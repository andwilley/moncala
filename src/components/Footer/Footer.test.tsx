import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './Footer';

describe('<Footer> test a basic render output', () => {
  it('should render the notepad', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
