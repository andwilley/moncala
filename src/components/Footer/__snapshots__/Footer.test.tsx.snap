// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer> test a basic render output should render the notepad 1`] = `
<div
  className="Footer h2"
>
  <p
    className="mt2 f7 fr"
  >
    Created By: Drew Willey. For: Mon Cala Project Application.
  </p>
</div>
`;
